pagopa/pn-frontend
# PN-8797-fe-pg-nel-portale-pg-sostituire-lemail-di-assistenza-nellheader-con-il-link-assistenza

## Short description
The behavior of the support link in PG has been modified for logged-in users.

## List of changes proposed in this pull request
- Added the link `${SELFCARE_BASE_URL}/assistenza` to `handleAssistanceClick()` once the user is logged in.
- Added on CDN config file `/assistenza`.

## How to test
Simply run `yarn start` for PG and check if the assistance link redirects to a different path after logging in.
